server/plugin/api.go

```diff
@@ -1380,6 +1401,102 @@ func (p *Plugin) getRepositoryListByOrg(c context.Context, ghInfo *GitHubUserInf
 	return allRepos, http.StatusOK, nil
 }
 
+func (p *Plugin) getOrganizations(c *UserContext, w http.ResponseWriter, r *http.Request) {
+	var allOrgs []*github.Organization
+	org := p.getConfiguration().GitHubOrg
+
+	if org == "" {
+		includeLoggedInUser := r.URL.Query().Get("includeLoggedInUser")
+		if includeLoggedInUser == "true" {
+			allOrgs = append(allOrgs, &github.Organization{Login: &c.GHInfo.GitHubUsername})
+		}
+		githubClient := p.githubConnectUser(c.Context.Ctx, c.GHInfo)
+		orgList, err := getOrganizationList(c.Ctx, "", githubClient, github.ListOptions{PerPage: 50})
+		if err != nil {
+			c.Log.WithError(err).Warnf("Failed to list organizations")
```

Comment:
Shouldn't this be an error level log?  It is returning a 500 error on the request.

Reply:
@cyrusjc is not actively working on this PR so I will be fixing these issuse

Reply:
@wiggin77 I have fixed these issues at  multiple places